generateDockerfiles.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const mustache = require('mustache');
+const config = require('./config.json');
+
+const templateFile = fs.readFileSync('Dockerfile.template', 'utf8');
+mustache.parse(templateFile);
+
+for (const dockerfile of config.dockerfiles) {
```

Comment:
It is not a big deal in this case but typically with modules node executes directly, it's preferred if you surround with something like this (similar to python's `if __name__ == '__main__'`):

https://nodejs.org/docs/latest-v8.x/api/modules.html#modules_accessing_the_main_module

```

function run() {
    ...
}

if (require.main === module) {
    run();
}
```

Maybe a simple input validation on `dockerfile` properties would be helpful. (at the very least check if `['tag', 'from']` exist)